cmd/server-main.go

```diff
@@ -591,8 +591,11 @@ func serverMain(ctx *cli.Context) {
 		}
 	}
 
+	// initialize the new disk cache objects.
 	if globalCacheConfig.Enabled {
-		// initialize the new disk cache objects.
+		if globalIsErasure {
+			logStartupMessage(color.Yellow("WARNING: Disk caching is deprecated for single/multi drive MinIO setups. Please migrate to using MinIO S3 gateway instead of disk caching"))
+		}
```

Comment:
```suggestion
		logStartupMessage(color.Yellow("WARNING: Disk caching is deprecated for single/multi drive MinIO setups. Please migrate to using MinIO S3 gateway instead of disk caching"))
```